governing-documents/security.md

```diff
@@ -0,0 +1,266 @@
+
+This policy borrows heavily from the recommendations
+of the OpenSSF Vulnerability Disclosure working group.  For up-to-date
+information on the latest recommendations related to vulnerability disclosures,
+please visit the github of that working group, available at
+https://github.com/ossf/wg-vulnerability-disclosures.
+
+## Outline:
+
+This document has the following sections:
+1. [What is a Vulnerability Disclosure Policy?](#what-is-a-vulnerability-disclosure-policy)
+2. [Vulnerability Disclosure Processes and Associated rules](#vulnerability-disclosure-process-and-associated-rules)
+3. [“People” Infrastructure](#people-infrastructure)
+4. [Report Intakes](#report-intakes)
+5. [CNA/CVE Reporting](#cnacve-reporting)
+6. [Embargo List](#embargo-list)
+7. [(Github) Security Advisories](#github-security-advisories)
+8. [Private Patch Deployment Infrastructure](#private-patch-deployment-infrastructure)
+
+## What Is a Vulnerability Disclosure Policy?
+
+No piece of software is perfect.  All software (at least, all software of a
+certain size and complexity) has bugs.  In open source development, members of
+the community or the public find bugs and report them to the project.  A
+vulnerability disclosure policy explains how this process  functions from the
+perspective of the project.
+
+This vulnerability disclosure policy explains the rules and guidelines for the
+XYZ Project.  It is intended to act as both a reference for outsiders–including
+both bug reporters and those looking for information on the project’s security
+practices–as well as a set of rules that maintainers and contributors have
+agreed to follow.
+
+## Vulnerability Disclosure Process and Associated Rules
+
+All Hyperledger projects, including this project, follow the associated process
+and rules for vulnerability disclosures.  We note that this outline is derived
+from the OpenSSF,
+https://github.com/ossf/oss-vulnerability-guide/blob/main/maintainer-guide.md.
+
+Each project will have a security team (for more details, see the section on
+people infrastructure below).  The security team will be comprised of
+maintainers or contributors to the project who are knowledgeable about
+security and is responsible for responding to and helping to fix security
+vulnerabilities.
+
+The security team (see below for details) for this project will do the
+following for each reported vulnerability:
+1. Acknowledge receipt (see the section on “Report Intakes” for more) of the
+issue to the reporter within 2 business days.
+2. Assess the issue.  Engage with the reporter to ask any outstanding questions
+about the report and how to reproduce it.   If the report is not considered a
+vulnerability, then the reporter should be informed and this process can be
+halted. If the report is still a regular bug (just not a security
+vulnerability), the reporter should be informed (if necessary) of the regular
+process for reporting bugs, available <where your project has regular bug reports>.
```

Comment:
```suggestion
process for reporting bugs, available \<where your project has regular bug reports\>.
```
Not sure if this is the correct formatting. The portion in the brackets do not show up in the markdown.